test/system/account.test.ts

```diff
@@ -1,7 +1,7 @@
-import * as battleScene from "#app/battle-scene";
 import { api } from "#app/plugins/api/api";
 import { describe, expect, it, vi } from "vitest";
 import { initLoggedInUser, loggedInUser, updateUserInfo } from "#app/account";
+import * as bypassLogin from "#app/constants";
```

Comment:
```suggestion
import * as constants from "#app/constants";
```
And change the other instances as needed.